### DISCUSSION TITLE
withExternalParentOf not finding indirect parents

### DISCUSSION THREAD

**Discussion by OscarSpruit** [LemonAppDev/konsist#1242] [Report a Problem]
### Tell us what version of Konsist you use.

0.15.1

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I am trying to select all classes that inherit from `android.view.View` with the code below. In my project this doesn't select any class, because no class directly inherits from `View`. When I change `View` to for example `LinearLayout` some classes show up as they inherit this directly. I have tried `withParentOf()` as well, but this has the same behaviour. 

```Kotlin
Konsist
    .scopeFromProject()
    .classes(includeNested = true)
    .withExternalParentOf(
        View::class,
        indirectParents = true,
    )
    .print()
```

I noticed that `withParentOf()` does work correctly when the indirect class is part of your projects codebase, so that's why I'm assuming `withExternalParentOf` has a bug.

### Actual behaviour

`withExternalParentOf` does not find indirect parents of external classes.

### Expected behaviour

`withExternalParentOf` does find indirect parents of external classes.

**Comment by sardmendoza** [comment]
I am also encountering the same problem. withExternalParentOf works as long as it is a direct parent but fails for indirect ones.
